editor/test/editor/fs_test.clj

```diff
@@ -666,3 +666,12 @@
         (is (= ["destination.txt"] (test-util/file-tree dir)))
         (is (= src-contents (slurp tgt)))))))
 
+(deftest below-directory-test
+  (test-util/with-temp-dir! project-dir
+    (is (true? (fs/below-directory? (io/file project-dir "file.txt") project-dir)))
```

Comment:
Why `true?` here?

Reply:
I wanted to ensure it returns a boolean value, not just a "truthy" one. Is that what you meant?

Reply:
Yeah.